apps/builder/app/builder/features/workspace/canvas-tools/outline/use-outline.ts

```diff
@@ -0,0 +1,23 @@
+import { useStore } from "@nanostores/react";
+import type { Instance } from "@webstudio-is/sdk";
+import type { Atom } from "nanostores";
+import { scaleStore } from "~/builder/shared/nano-states";
+
+export const useOutline = (
```

Comment:
Can be computed

Reply:
Ah, got it. You pass different stores with same shape.

Reply:
yes